xml/System.Globalization/SortVersion.xml

```diff
@@ -53,14 +56,20 @@
 |[!INCLUDE[net_v45](~/includes/net-v45-md.md)] and later versions of the .NET Framework|[!INCLUDE[win8](~/includes/win8-md.md)] and later Windows operating system versions|Unicode 6.0|  
   
  On [!INCLUDE[win8](~/includes/win8-md.md)], because the version of Unicode used in string comparison and ordering depends on the version of the operating system, the results of string comparison may differ even for applications that run on a specific version of the .NET Framework.  
-  
- The <xref:System.Globalization.SortVersion> class provides information about the Unicode version used by the .NET Framework for string comparison and ordering. It enables developers to write applications that can detect and successfully handle changes in the version of Unicode that is used to compare and sort an application's strings.  
+
+### Sorting and string comparison in .NET Core
+
+All versions of .NET Core support a single version of the Unicode Standard, [The Unicode Standard, Version 8.0.0](https://www.unicode.org/versions/Unicode8.0.0/). [International Components for Unicode](http://site.icu-project.org/) libraries provide the implementation for comparison and sorting APIs.
```

Comment:
>ll versions of .NET Core support a single version of the Unicode Standard, [The Unicode Standard, Version 8.0.0] [](start = 1, length = 112)

this can be confusing because at the end we depend on the OS for the string comparisons. so it depends what the OS support which version of the Unicode to perform the sort comparisons.